book/http_fundamentals.rst

```diff
@@ -41,9 +41,6 @@ Symfony is built from the ground up around that reality. Whether you realize
 it or not, HTTP is something you use every day. With Symfony, you'll learn
 how to master it.
 
-.. index::
-   single: HTTP; Request-response paradigm
```

Comment:
@paxyknox some people are reviewing lots of PRs each days, so they may sound a bit to the point. We are very happy with your PR and having 10 wrong changes in a PR containing 1000 changes is still not bad :wink: 

In this case, indexes are used on the index of the documentation to redirect people to sections of the docs: http://symfony.com/doc/current/genindex.html While I can't see much references to it, I think we should keep the indexes as they were for now.